var/spack/repos/builtin/packages/xrootd/package.py

```diff
@@ -15,6 +15,7 @@ class Xrootd(CMakePackage):
     url = "http://xrootd.org/download/v5.3.1/xrootd-5.3.1.tar.gz"
     list_url = "https://xrootd.slac.stanford.edu/dload.html"
```

Comment:
We generally prefer that fetch (of a version) use the `https` protocol for downloads.  You could change the `url` and remove the `list_url` as follows and all of the checksums are still valid:

```suggestion
    url = "https://xrootd.slac.stanford.edu/download/v5.5.1/xrootd-5.5.1.tar.gz"
```